src/main/java/com/telefonica/iot/cygnus/backends/mongo/MongoBackend.java

```diff
@@ -200,7 +201,8 @@ public void insertContextDataAggregated(String dbName, String collectionName, lo
      * @param attrValue
      * @param resolution
      */
-    private void insertContextDataAggregatedForResoultion(String dbName, String collectionName, GregorianCalendar calendar,
+    private void insertContextDataAggregatedForResoultion(String dbName, String collectionName,
+                                                          GregorianCalendar calendar,
```

Comment:
Yes. According to the max line length rule (120 characters) specified in the checkstyle guidelines, I splitted the existing row, in order to pass the check (even if  the method don't affect my feature).